.gitignore

```diff
@@ -24,3 +24,4 @@ docs/examples/rpcclient/rpcclient
 wiki
 dex/testing/loadbot/loadbot
 bin/
+localized_html/
```

Comment:
Why not check these in?  Would be nice to have one less thing in our workflow.

Reply:
Done. Now I'm thinking about how we can check these in automated testing.